spec/mailers/previews/order_preview.rb

```diff
@@ -0,0 +1,10 @@
+# Preview all emails at http://localhost:3000/rails/mailers/order
+# loads FactoryBot to use build strategy
+class OrderPreview < ActionMailer::Preview
+  include FactoryBot::Syntax::Methods
+
+  def order_delivered_email
+    order = build(:order)
```

Comment:
how bout:
order = FactoryBot.build(:order)
you do not have to include FactoryBot then

Reply:
That's good idea. I have not though of that.